### DISCUSSION TITLE
Welcome to plenti Discussions!

### DISCUSSION THREAD

**Discussion by jimafisk** [plentico/plenti#248] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jamestagal** [comment]
Hello @jimafisk and all,
Thanks for creating this collaboration/sharing space so we can discuss all about this awesome project, Plenti. 😊

I'm Benjamin (or Ben for short), a teacher/education designer working at the [Canberra Institute of Technology ](https://cit.edu.au/)(Canberra, Australia) and in my spare time I like to learn about web technologies.  This time last year, I came across one of Jim's Jamstack Boston videos on Youtube and got interested in Plenti. I then created my first blog site [EdTech Designer](https://edtechdesigner.io/) using Roberto Rivera's (@roobyz) cool [Plenti theme](https://plenti.co/themes) ```Compendium``` and was hooked so as a result started sponsoring the project soon after.  

To learn more about Plenti, I watched the [Plenti Bigsping theme development](https://www.youtube.com/watch?v=wyNC7R_VVyQ&list=PLbWvcwWtuDm12y3Hye6oKDwI2gAS0ccHW) video series Jim created on converting a Hugo theme to Plenti. Then I took an opportunity to reach out to Jim and offered to help build Plenti themes as a way for me to dive into real world situations and learn more about the underlying tech, particularly Javascript/Svelte. Fast forward a few months, I have now contributed a couple of themes and intend to continue this work (now a passion of mine) and build on my knowledge and understanding of the platform. I love learning and think I learn something new everyday working on this project, thanks to the tremendous help I get from @jimafisk. Much appreciate. 😃

I am excited about Plenti and I am keen to connect with likeminded people here. So I look forward to the discussions! Here's to  another great year of Plenti development in 2023. 
Regards,
Ben
